src/rest/RESTManager.js

```diff
@@ -18,9 +18,10 @@ class RESTManager {
     this.globalReset = null;
     this.globalDelay = null;
     if (client.options.restSweepInterval > 0) {
-      client.setInterval(() => {
+      const interval = client.setInterval(() => {
```

Comment:
Couldn't you just call `unref` on the returned interval?
Since the variable isn't used otherwise.

Reply:
Eh, it's more readable this way... 🤷‍♂️